.buildkite/prepare-claims.yml

```diff
@@ -47,37 +47,40 @@ steps:
         --output-validator-meta-collection "$$snapshot_dir/validators.json" \
         --output-stake-meta-collection "$$snapshot_dir/stakes.json"
 
+  - label: ":floppy_disk: :arrow_left: :cloud: Fetch past validator data"
+    env:
+      gs_bucket: gs://marinade-validator-bonds-mainnet
+    commands:
+    - 'snapshot_dir=$(buildkite-agent meta-data get snapshot_dir)'
+    - 'epoch=$(buildkite-agent meta-data get epoch)'
+    - 'previous_epoch=$((epoch - 1))'
```

Comment:
When the previous epoch did not generate the `validators.json` - is it something we expect it can happen or it's just something that "won't happen"? If possibly happens it seems a manual intervention is needed, could that be mentioned somewhere what could be the process?

Reply:
This is expected when we are starting the process in the past to backfill data or for the first ever run of this updated pipeline. Previous data not existing is therefore error only sometimes. For cases other than backfill, this will not happen as it would mean the previous run did not succeed and it was not fixed (which is unexpected).

All that is to be done is re-run the failing pipeline, not sure this is worth documenting.

Reply:
Ok, got it, thanks I did not have this context.